### PR TITLE
Build pepabo engage page

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build engage page at `/engage/case-study-gmo-pepabo`
- Add to `/engage/index.html`
- Create [social banner](https://assets.ubuntu.com/v1/528cf2ab-facebook_and_linkedin_banner+%283%29.jpeg) 

## QA

- Check out this feature branch
- Run the site using the command ./run serve or dotrun
- View the site locally in your web browser at: [http://0.0.0.0:8012/engage](http://0.0.0.0:8012/engage) to check GMO pepabo added and check engage page at [http://0.0.0.0:8012/engage/case-study-gmo-pepabo](http://0.0.0.0:8012/engage/case-study-gmo-pepabo) matches [copy doc](https://docs.google.com/document/d/1TewivXyiRIQyz8m4IXg1o1gC7nsZAxrfcrB32szE8jI/edit#) and [design](https://github.com/canonical-web-and-design/web-squad/issues/3055)

## Issue / Card

Fixes [#8363](https://github.com/canonical-web-and-design/ubuntu.com/issues/8363)

### CODE REVIEW

**Comment by SirSamTumless** [comment]
Waiting on PDF

File: templates/engage/case-study-gmo-pepabo.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% extends "_base/base.html" %}
+
+{% block title %}GMOペパボについて{% endblock %}
+
+{% block meta_description %}GMOペパボがカーネルのライブパッチによって運用コストと2,500時間の手作業を削減{% endblock %}
+
+{% block meta_image %}{% endblock %}
```
Comment (sowasred2012) [inline templates/engage/case-study-gmo-pepabo.html:7]: This is missing the meta image given [here](https://github.com/canonical-web-and-design/web-squad/issues/3055#issuecomment-676479694) - you may need to upload it to the asset manager, not sure if it already has been.

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
LGTM, just a missing meta image

**Review by sowasred2012 — APPROVED** [review]
LGTM!

**Comment by AmberCharitos** [comment]
@SirSamTumless @bethcollins92 the brief was for this engage page to also be in Japanese - it looks as though it is in English - [here](https://docs.google.com/document/d/1TewivXyiRIQyz8m4IXg1o1gC7nsZAxrfcrB32szE8jI/edit) is the copy doc for the Japanese version of the landing page

**Comment by SirSamTumless** [comment]
@AmberCharitos It is in jP?

**Comment by AmberCharitos** [comment]
@SirSamTumless my browser auto translated to English (wow that was silly of me) - sorry had a moment - thank you!!

**Comment by AmberCharitos** [comment]
@bethcollins92 @SirSamTumless, we've had some edits requested from GMO Pepabo on this page.
 
1) An update to the first paragraph on the engage page - updated in [this](https://docs.google.com/document/d/1TewivXyiRIQyz8m4IXg1o1gC7nsZAxrfcrB32szE8jI/edit#) doc 
2) The logo used has been changed colour and they would like it updated to one including their brand colouring - I am waiting for them to send me a copy of this logo with a black background. This is something they would also like changed in the uk version. 
3) The quote that is pulled out has been cut with the orange close bracket - could this be shifted so as not to obscure the text? 

Sorry for all this!

**Pull request merged**